WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -93,6 +93,8 @@ struct EditableTextRow: ImmuTableRow {
     func configureCell(_ cell: UITableViewCell) {
         cell.textLabel?.text = title
         cell.detailTextLabel?.text = value
+        cell.accessibilityLabel = title
+        cell.accessibilityHint = value
```

Comment:
I wonder if editable rows should also say something like 'tap to edit' 🤔 Although we would also want to check if they were enabled before doing so.

Reply:
Nice suggestion! 😄 I’ve made an update for this using isUserInteractionEnabled to check state for whether enable() or disable() was called